### PR TITLE
Update doc to include windows ssh instructions

### PR DESCRIPTION
**Pull request by jbritt1**
Ref: #7044

### CODE REVIEW

**Comment by jbritt1** [comment]
Tagging @department-of-veterans-affairs/vsp-operations

File: platform/engineering/internal-tools.md
Comment (ricetj) [inline platform/engineering/internal-tools.md]: Thank you for adding this.

**Review by ricetj — APPROVED** [review]
Looks good to me and thanks for adding this.

**Comment by jbritt1** [comment]
Good to merge or perhaps should wait on another set of eyes / review you think @ricetj?

**Comment by ricetj** [comment]
> Good to merge or perhaps should wait on another set of eyes / review you think @ricetj?

Yeah, it would be good to have another set of eyes.

**Review by nathanhruby — APPROVED** [review]

**Pull request merged**
